compiler/src/iree/compiler/Codegen/Dialect/GPU/TransformExtensions/test/distribute_multi_mma.mlir

```diff
@@ -29,7 +29,7 @@ module attributes { transform.with_named_sequence } {
 }
 
 // CHECK-DAG:   #[[$MAP:.+]]  = affine_map<(d0) -> (d0 mod 16)>
-// CHECK-DAG:   #[[$MAP1:.+]] = affine_map<(d0) -> ((d0 floordiv 16) mod 4)>
+// CHECK-DAG:   #[[$MAP1:.+]] = affine_map<(d0) -> ((d0 floordiv 16) * 4 - ((d0 floordiv 16) floordiv 4) * 16)>
```

Comment:
This looks weird because it is the result of canonicalization. It's `affine_map<(d0) -> (((d0 floordiv 16) mod 4) * 4)>` before running the canonicalizer.